pkg/constants/constants.go

```diff
@@ -0,0 +1,16 @@
+package constants
+
+const (
+	SingletonName              = "instance"
+	OpenshiftNS                = "openshift-logging"
+	ProxyName                  = "cluster"
+	TrustedCABundleKey         = "ca-bundle.crt"
+	TrustedCABundleMountDir    = "/etc/pki/ca-trust/extracted/pem/"
+	TrustedCABundleMountFile   = "tls-ca-bundle.pem"
+	InjectTrustedCABundleLabel = "config.openshift.io/inject-trusted-cabundle"
+	TrustedCABundleHashName    = "logging.openshift.io/hash"
+	FluentdTrustedCAName       = "fluentd-trusted-ca-bundle"
```

Comment:
unnecessary in EO

Reply:
I see some of them are used to create Kibana deployment